cpp/tests/grouped_rolling/grouped_rolling_test.cpp

```diff
@@ -552,11 +552,11 @@ TYPED_TEST(GroupedRollingTest, SimplePartitionedStaticWindowsWithGroupKeys)
   // 2 grouping keys, with effectively 3 groups of at most 4 rows each:
   //   1. key_0 {0, 0, 0, ...0}
   //   2. key_1 {0, 0, 0, 0, 1, 1, 1, 1, 2, 2}
-  std::vector<TypeParam> key_0_vec(DATA_SIZE, 0);
+  std::vector<TypeParam> key_0_vec(DATA_SIZE, TypeParam(0));
```

Comment:
wouldn't you have to include the duration types for this typed test (`GroupedRollingTest`) for these changes to be necessary?

Reply:
added.